learning/tour-of-beam/frontend/lib/pages/welcome/screen.dart

```diff
@@ -65,15 +65,23 @@ class _SdkSelection extends StatelessWidget {
 
   @override
   Widget build(BuildContext context) {
-    return ColoredBox(
+    return Container(
+      constraints: BoxConstraints(
+        minHeight: MediaQuery.of(context).size.height -
```

Comment:
Explicit sizes should be the last resort. Can this be done with this?

For two columns:
```
WelcomeScreen
+- PageContainer
   +- SingleChildScrollView
      +- Stack
         +- Row
            +- Left column without laptop but with blank container with the height of the laptop
            +- Right column
         +- Laptop sticked to the bottom
```

For one column:
```
WelcomeScreen
+- PageContainer
   +- SingleChildScrollView
      +- Column
         +- Left column without laptop but with blank container
         +- Laptop
         +- Right column
```

Reply:
https://user-images.githubusercontent.com/31556582/187871423-d79c544d-e0a3-43a4-9bbd-0d5f8d20ca8b.mov

Reply:
in that case, the laptop is independent of the left column's size.

Reply:
Can the `Positioned` contain something like this?

```dart
Row(
  children: [
    Expanded(/* laptop */),
    Expanded(/* empty SizedBox */),
  ],
),
```

The two `Expanded`s will split the full width in half. And the laptop will occupy the full width of its half.
If it works, we may need to put this under the content so that the right half does not make the content unclickable.

Reply:
<img width="592" alt="Screen Shot 2022-09-01 at 14 53 48" src="https://user-images.githubusercontent.com/31556582/187874174-5d06b4e7-21d6-4318-9b56-c53eb7ed4f45.png">
<img width="1428" alt="Screen Shot 2022-09-01 at 14 54 11" src="https://user-images.githubusercontent.com/31556582/187874236-598644bf-4816-4e51-ac74-7947399a7dbf.png">

Reply:
the height of the laptop is completely dependent on the height of the sized box inside of the left column.

Reply:
OK, let's go with `MediaQuery` solution for now.